core/src/main/java/org/arquillian/smart/testing/SchemaGenerator.java

```diff
@@ -0,0 +1,89 @@
+package org.arquillian.smart.testing;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+class SchemaGenerator {
```

Comment:
if the SchemaGenerator is related to configuration, then move it there - into the same package